oak_containers_launcher/src/lib.rs

```diff
@@ -74,15 +75,16 @@ const VM_START_TIMEOUT: u64 = 300;
 #[group(skip)]
 pub struct Args {
     #[arg(long, required = true, value_parser = path_exists,)]
-    system_image: std::path::PathBuf,
+    pub system_image: std::path::PathBuf,
     #[arg(long, required = true, value_parser = path_exists,)]
-    container_bundle: std::path::PathBuf,
+    pub container_bundle: std::path::PathBuf,
     #[arg(long, value_parser = path_exists,)]
-    application_config: Option<std::path::PathBuf>,
+    pub application_config: Option<std::path::PathBuf>,
     #[command(flatten)]
-    qemu_params: qemu::Params,
+    pub qemu_params: qemu::Params,
 }
 
+#[cfg(feature = "test-args")]
```

Comment:
I don't think it's worth adding this feature -- just making them pub seems fine to me. @ipetr0v @conradgrobler WDYT?

Reply:
The feature flag here is because of the use of WORKSPACE_ROOT to generate the defaults, which causes compilation failures from an external depender.

Another solution is to just move these arg constructors into the hello world integration test code.

Reply:
Another option might be to pass the root path as an argument to the default functions and then only use `WORKSPACE_ROOT` from inside the tests. I don't have a strong preference between any of these options though.

Reply:
That's a great idea, adding a new feature does feel like a bit much.